integration-cli/docker_cli_save_load_unix_test.go

```diff
@@ -29,9 +29,8 @@ func (s *DockerSuite) TestSaveAndLoadRepoStdout(c *check.C) {
 	saveCmd := exec.Command(dockerBinary, "save", repoName)
 	saveCmd.Stdout = tmpFile
 
-	if _, err = runCommand(saveCmd); err != nil {
-		c.Fatalf("failed to save repo: %v", err)
-	}
+	_, err = runCommand(saveCmd)
```

Comment:
@cpuguy83 the thing is that `dockerCmd` doesn't let you setup the `Stdout` of the command (which is done there).